functions/airtable.js

```diff
@@ -9,52 +9,94 @@ const base = airtable.base(functions.config().airtable.base_id);
 
 const INTAKE_CONTACTS = functions.config().airtable.intake_contacts_table;
 const INTAKE_MESSAGES = functions.config().airtable.intake_messages_table;
+// const INBOUND = functions.config().airtable.inbound_table;
+const INTAKE = functions.config().airtable.intake_table;
 
-module.exports = {
-
-  getPhoneNumberId: (phoneNumber) => (
-    base(INTAKE_CONTACTS).select({
-      maxRecords: 1,
-      filterByFormula: `{phone_number} = "${phoneNumber}"`
-    }).firstPage().then(records => {
-      if (records[0]) {
-        return records;
-      } else {
-        return base(INTAKE_CONTACTS).create([
-          {
-            fields: {
-              phone_number: phoneNumber,
-              intake_status: 'intake needed'
-            }
+function getPhoneNumberId(phoneNumber) {
+  return base(INTAKE_CONTACTS).select({
+    maxRecords: 1,
+    filterByFormula: `{phone_number} = "${phoneNumber}"`
+  }).firstPage().then(records => {
+    if (records[0]) {
+      return records;
+    } else {
+      return base(INTAKE_CONTACTS).create([
+        {
+          fields: {
+            phone_number: phoneNumber,
+            intake_status: 'intake needed'
           }
-        ]);
-      }
-    }).then(records => records[0].id)
-  ),
-
-  createMessage: (phoneNumberId, message) => (
-    base(INTAKE_MESSAGES).create([
-      {
-        fields: {
-          type: 'SMS',
-          phone_number: [phoneNumberId],
-          message: message,
         }
-      },
-    ])
-  ),
-
-  createVoicemail: (phoneNumberId, recordingUrl, message) => (
-    base(INTAKE_MESSAGES).create([
-      {
-        fields: {
-          type: 'Voicemail',
-          phone_number: [phoneNumberId],
-          recording_url: recordingUrl, 
-          message: message,
-        }
-      },
-    ])
+      ]);
+    }
+  }).then(records => records[0].id)
+}
+
+function createMessage(phoneNumberId, message) {
+  return base(INTAKE_MESSAGES).create([
+    {
+      fields: {
+        type: 'SMS',
+        phone_number: [phoneNumberId],
+        message: message,
+      }
+    },
+  ])
+}
+
+function createVoicemail(phoneNumberId, recordingUrl, message) {
+  base(INTAKE_MESSAGES).create([
+    {
+      fields: {
+        type: 'Voicemail',
+        phone_number: [phoneNumberId],
+        recording_url: recordingUrl,
+        message: message,
+      }
+    },
+  ])
+}
+
+async function getAllIntakeTickets() {
+  const query = base(INTAKE).select()
+  const records = await query.all()
+
+  return records.map(rec => [rec.id, rec.fields])
```

Comment:
would rather this be an object in the shape of `{[id]: fields}` but this is ok for now